cmd/server/cadence/server.go

```diff
@@ -58,9 +58,9 @@ type (
 	}
 )
 
-// newServer returns a new instance of a daemon
+// NewServer returns a new instance of a daemon
 // that represents a cadence service
-func newServer(service string, cfg *config.Config) common.Daemon {
+func NewServer(service string, cfg *config.Config) common.Daemon {
```

Comment:
Any reason why this became public? I do not see this being used outside of this package.

Reply:
Yeah I will change it back. Initially I was writing `main_test.go` but then I moved it to  `server_test.go`